pkg/autoscaling/types.go

```diff
@@ -138,3 +148,38 @@ func (t *TiDBInfo) getLabelValue(key string) string {
 func (t *TiDBInfo) getLabels() map[string]string {
 	return t.Labels
 }
+
+type tidbInformerImpl struct {
+	etcdClient *clientv3.Client
+}
+
+func newTidbInformer(client *clientv3.Client) tidbInformer {
+	return &tidbInformerImpl{
+		etcdClient: client,
+	}
+}
+
+func (informer *tidbInformerImpl) GetTiDB(address string) (*TiDBInfo, error) {
+	key := fmt.Sprintf("/topology/tidb/%s/info", address)
+	resp, err := etcdutil.EtcdKVGet(informer.etcdClient, key)
+	if err != nil {
+		return nil, err
+	}
+	if resp.Count < 1 {
+		err := fmt.Errorf("resp loaded for tidb[%s] is empty", address)
```

Comment:
There is no need to define a new error. We can directly output what we wanna show in the log.

Reply:
updated.